### PR TITLE
Fix #1: Only show/process for "default" group

### LINKED ISSUE #1: "Open in Iodide" button should not be available for queries accessing restricted datasets (e.g. search)
This is a copy of [redash-stmo issue #59](https://github.com/mozilla/redash-stmo/issues/59). I'm moving that issue here because the Iodide plugin now lives here.

In that issue, @wlach said:

> One mitigation item mentioned by [the RRA](https://bugzilla.mozilla.org/show_bug.cgi?id=1583543) we did on the Iodide server was that the "open in iodide" button should not be made available for queries that use a restricted dataset (e.g. search). 
>
> If this is relatively easy, we should probably just do it. /cc @jezdez @openjck 

For more information, see the discussion in [redash-stmo issue #59](https://github.com/mozilla/redash-stmo/issues/59).

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by openjck** [comment]
@jezdez I was wondering if you could help me review this.

I also have a question. On STMO, is the *default* group used for data that all employees should have access to? I assume that's the case, but I'd like to confirm that before I reach out to the author of the RRA to ask him what he thinks about this fix.

**Review by emtwo — APPROVED** [review]
This seems like a reasonable and important change to me and it looks good. We were blocking moving forward with Redash work until Jannis comes back so you may need to check with him before merging this.

**Comment by openjck** [comment]
Sounds good. I'll ping Jannis early next week when things have settled down for him. Thank you so much for the review!

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+2020.1.1
+--------
+
+:date: 2020-01-17
```
Comment (jezdez) [inline CHANGELOG.rst]: This would need an update

File: src/redash_iodide/__init__.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 # file, you can obtain one at http://mozilla.org/MPL/2.0/.
 """Redash extensions for Iodide by Mozilla"""
 
-__version__ = "2019.11.0"
+__version__ = "2020.1.1"
```
Comment (jezdez) [inline src/redash_iodide/__init__.py]: Also needs an update to follow CalVer

File: src/redash_iodide/explore/extension.py [hunk 3]
```diff
@@ -19,8 +19,23 @@
 class IodideNotebookResource(BaseResource):
     TEMPLATE_PATH = os.path.join(os.path.dirname(__file__), "iodide-notebook.iomd.j2")
 
+    # The ID of the "default" group in the Redash access control system
+    default_group_id = 2
```
Comment (jezdez) [inline src/redash_iodide/explore/extension.py]: This should be a setting

File: src/redash_iodide/explore/extension.py [hunk 4]
```diff
@@ -50,6 +65,6 @@ def extension(app):
         app,
         IodideNotebookResource,
         "/api/integrations/iodide/<query_id>/notebook",
-        endpoint="idodide_notebook",
+        endpoint="iodide_notebook",
```
Comment (jezdez) [inline src/redash_iodide/explore/extension.py:65]: Nice catch!

**Review by jezdez — APPROVED** [review]
Only minor things

File: src/redash_iodide/settings.py [hunk 5]
```diff
@@ -34,6 +34,6 @@ def extension(app):
         app,
         IodideSettingsResource,
         "/api/integrations/iodide/settings",
-        endpoint="idodide_settings",
+        endpoint="iodide_settings",
```
Comment (jezdez) [inline src/redash_iodide/settings.py:37]: D'oh!

File: src/redash_iodide/explore/extension.py (hunk 3 above)
Comment (openjck) [reply src/redash_iodide/explore/extension.py]: 👍

**Comment by openjck** [comment]
Updated! Thank you, Jannis. 😃

**Comment by openjck** [comment]
I'm not exactly sure why the tests are failing. Unfortunately, I can no longer test it locally because I lost access to the environment variables that are needed to run this locally:

* `REDASH_IODIDE_URL`
* `REDASH_IODIDE_NOTEBOOK_API_URL`
* `REDASH_IODIDE_AUTH_TOKEN`
* `REDASH_IODIDE_REDASH_DEFAULT_GROUP_ID`

The tests were passing before the final one was added. Perhaps we need to add that in CI? Unfortunately I don't have access to our CI settings.

**Comment by openjck** [comment]
Updated the version and release date

**Comment by jezdez** [comment]
Thank you @openjck, I've fixed the failing test! It wasn't mocking the default group ID correctly in the tests and so I fixed that by removing the need for having to hard-code the default group ID in the first place.

**Comment by openjck** [comment]
Thank you, Jannis! Excited to see this land.

**Pull request merged**